packages/terra-clinical-result/src/ClinicalResult.jsx

```diff
@@ -71,11 +70,6 @@ const propTypes = {
    * Used by Flowsheet Result Cell to hide icons because it displays them in different positions.
    */
   hideAccessoryDisplays: PropTypes.bool,
-  /**
-   * @private
-   * The intl object to be injected for translations.
-   */
-  intl: intlShape.isRequired,
```

Comment:
I thought this was okay to remove passively since it's a private prop. Alternatively, I could leave it as deprecated. It wasn't being used in the code anywhere for this and a couple of other components in this package.

Reply:
I agree, this is ok.